server/tso/global_allocator.go

```diff
@@ -50,6 +52,9 @@ type Allocator interface {
 	Reset()
 }
 
+// TODO: make enableGlobalTSOEstimation become a config field.
+const enableGlobalTSOEstimation = true
```

Comment:
Seem we can't disable it once we start the cluster. It's better to remove it until we add a real switch.